bot/utils/__init__.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
+from typing import List
+
+import discord
+from discord.ext.commands import BadArgument, Context
+
+from bot.pagination import LinePaginator
+
+
+async def disambiguate(
+        ctx: Context, entries: List[str], *, timeout: float = 30,
+        per_page: int = 20, empty: bool = False, embed: discord.Embed = None
+):
+    """
+    Has the user choose between multiple entries in case one could not be chosen automatically.
+    :param ctx: Context object from discord.py
+    :param entries: List of items for user to choose from
+    :param timeout: Number of seconds to wait before canceling disambiguation
+    :param per_page: Entries per embed page
+    :param empty: Whether the paginator should have an extra line between items
+    :param embed: The embed that the paginator will use.
+    :return: Users choice for correct entry.
```

Comment:
Since this appears to be a general utility, might be nice to document the `BadArgument` exception raised.